.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,38 +3,32 @@ description: File a bug report for Babelfish Website
 title: "[Bug]: "
 labels: ["bug", "triage"]
 assignees:
-  - amazon-auto
+  - 3manuek
+  - xaviersierra
+  - gopinathpai
+  - nasbyj
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thanks for taking the time to fill out this bug report.
   - type: input
     id: contact
     attributes:
       label: Contact Details
-      description: How can we get in touch with you if we need more info?
+      description: Please enter your mail address for contacting in the case of needing more information.
```

Comment:
How about:

Please provide an email address where we can contact you for more information.